frontend/packages/kubevirt-plugin/src/models/templates/vm-template-yaml.ts

```diff
@@ -10,20 +10,21 @@ metadata:
   name: vm-template-example
   labels:
     kubevirt.io/os: fedora27
-    miq.github.io/kubevirt-is-vm-template: 'true'
     template.kubevirt.io/type: vm
+    flavor.template.kubevirt.io/Medium: 'true'
+    os.template.kubevirt.io/fedora27: 'true'
+    vm.kubevirt.io/template-namespace: openshift
+    workload.template.kubevirt.io/server: 'true'
   annotations:
-    description: VM template example
-    iconClass: icon-fedora
-    tags: 'kubevirt,ocp,template,linux,virtualmachine'
+    name.os.template.kubevirt.io/fedora27: Fedora 27
 objects:
   - apiVersion: kubevirt.io/v1alpha3
     kind: VirtualMachine
     metadata:
-      creationTimestamp: null
       labels:
-        kubevirt-vm: 'vm-\${NAME}'
-        kubevirt.io/os: fedora27
+        app: '\${NAME}'
+        vm.kubevirt.io/template.revision: '1'
+        vm.kubevirt.io/template: fedora-server-medium
```

Comment:
this will not work for all environments - some have the version appended to the name

Reply:
This can change between kubevirt versions.

But impact is low, imo, as we focus on particular kubevirt version - i.e. in VirtualMachineModel and so we will need to maintain similar dependencies manually anyway.
Please correct me.

It would be better to react dynamically by fetching this name and additional details (like cpu) from the fedora template at runtime. It would help in the future releases, but I am not sure whether it's not overcomplicating things for now.

Reply:
Yes, we could reuse createTemplate request functions for that. It would be longterm stable solution.

We can add it manually right now, but it has to correspond to each version. We should have at least variable for that and should not forget to change it.

Although I think detection logic for the version should be fairly simple.

Reply:
@suomiy @mareklibra we can add it as a `parameter` in the `parameters` section ? for simplicity

Reply:
not sure if that is simpler - we would have to modify quite a few things to account for that.